config/deploy/dev.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'server_settings'
+
+server 'sdr-services-dev.stanford.edu', user: 'sdr2service', roles: %w{app}
+set :deploy_to, "/var/sdr2service/#{ENV['SDR_APP']}"
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
When `server_settings.rb` was first refactored out of these files, the refactor assumed that all the ENV values were already set.  I can't yet understand the impact of refactoring these files like this.